source/gcp/supported-services/cloud_audit_logs.rst

```diff
@@ -7,34 +7,76 @@
+
+* **Policy Denied** audit logs are recorded when a Google Cloud service denies access to a user or service account because of a security policy violation.
+
+Wazuh is able to collect and analyze those logs using the :ref:`GCP Pub/Sub integration <pubsub>`.
+
+Configure Google audit logs collection
+--------------------------------------
+
+To enable Google audit logs collection, it is necessary to first ingest the audit logs into a Pub/Sub topic defining a custom log router. To do that, visit the `Google Cloud Logging section  <https://console.cloud.google.com/logs/router>`_ and click on the ``CREATE SINK`` button.
+
+.. thumbnail:: ../../images/gcp/gcp-create-sink-button.png
     :align: center
     :width: 100%
 
-These logs can be filtered on Kibana by ``logName``:
+Then, provide a descriptive name for the sink and click on the ``NEXT`` button.
 
-.. thumbnail:: ../../images/gcp/kibana-gcp-log-filter.png
+.. thumbnail:: ../../images/gcp/gcp-sink-name.png
     :align: center
     :width: 100%
 
-* **Admin Activity** audit logs contain log entries for API calls or other administrative actions that modify the configuration or metadata of resources.
+Once the name for the sink is chosen, it is necessary to select the sink destination. As sink service, choose **Cloud Pub/Sub topic**, and then create or choose a topic to be used as destination. Then click on the ``NEXT`` button.
 
-.. thumbnail:: ../../images/gcp/gcp-admin-activity.png
+.. thumbnail:: ../../images/gcp/gcp-sink-destination.png
     :align: center
     :width: 100%
 
-* **Data Access** audit logs contain API calls that read the configuration or metadata of resources, as well as user-driven API calls that create, modify or read user-provided resource data.
+Finally, use the following query to collect all the audit logs from every project. It is possible to edit it to only collect audit logs from a particular project.
+
+.. code-block:: none
+
+    logName=~("projects/.*/logs/cloudaudit.googleapis.com%2F(activity|data_access|system_event|policy)")
+
+If it is not necessary to filter any logs out of the sink, click on the ``CREATE SINK`` button to create it.
+
+.. thumbnail:: ../../images/gcp/gcp-create-sink.png
+    :align: center
+    :width: 100%
+
+Once this process is finished, you can configure the :ref:`Wazuh GCP Pub/Sub integration <pubsub>` to process the audit logs of the selected resources as usual.
+
+
+Kibana dashboard visualization
+------------------------------
+
+After configuring the GCP Pub/Sub module to fetch the audit logs from Google Cloud, it is possible to visualize the alerts generated in the Kibana dashboard.
+
+.. thumbnail:: ../../images/gcp/gcp-overview.png
+    :align: center
+    :width: 100%
+
+Google Cloud logs can be filtered on Kibana by the ``data.gcp.logName`` field:
```

Comment:
```suggestion
#. Google Cloud logs can be filtered on Kibana by the ``data.gcp.logName`` field:
```

Reply:
I think the kibana section looks better without having tasks with only two items.